src/dbcp/extract/fips_tables.py

```diff
@@ -9,8 +9,13 @@
 
 import dbcp
 
+# originally from https://www2.census.gov/geo/tiger/TIGER2021/
+CENSUS_URI = "gs://dgm-archive/census/tl_2021_us_county.zip"
+TRIBAL_LANDS_URI = "gs://dgm-archive/census/tl_2021_us_aiannh.zip"
 
-def _extract_census_counties(census_uri: Path) -> pd.DataFrame:
+
+@lru_cache(maxsize=1)  # county boundaries are also used in some transform modules
+def _extract_census_counties(census_uri: str) -> pd.DataFrame:
```

Comment:
Why did you decide to add this cache decorator?

Reply:
I needed to load the census county geometries a second time in order to QA the lat/lon fields in the ACP data. The geometries take like 10 seconds to load, so I thought I'd cache it so we save that huuuuge amount of time